docs/website/docs/examples/chess_production/code/chess-snippets.py

```diff
@@ -0,0 +1,201 @@
+        stop=stop_after_attempt(5),
+        wait=wait_exponential(multiplier=1.5, min=4, max=10),
+        retry=retry_if_exception(retry_load(("extract", "load"))),
+        reraise=True,
+    )
+    def load_data_with_retry(data):
+        return pipeline.run(data)
+
+    # @@@DLT_SNIPPET_END markdown_retry
+
+    # @@@DLT_SNIPPET_START markdown_pipeline
+    __name__ = "__main__"  # @@@DLT_REMOVE
+    if __name__ == "__main__":
+        # create dlt pipeline
+        pipeline = dlt.pipeline(
+            pipeline_name="chess_pipeline",
+            destination="duckdb",
+            dataset_name="chess_data",
+            full_refresh=True,
+        )
+        max_players = 5
+        # get data for a few famous players
+        data = chess(chess_url="https://api.chess.com/pub/", max_players=max_players)
+        load_info = pipeline.run(data)
+        print(load_info)
+    # @@@DLT_SNIPPET_END markdown_pipeline
+
+    # @@@DLT_SNIPPET_START markdown_inspect
+        # see when load was started
+        print(f"Pipeline was started: {load_info.started_at}")
+        # print the information on the first load package and all jobs inside
+        print(f"First load package info: {load_info.load_packages[0]}")
+        # print the information on the first completed job in first load package
+        print(
+            f"First completed job info: {load_info.load_packages[0].jobs['completed_jobs'][0]}"
+        )
+
+    # @@@DLT_SNIPPET_END markdown_inspect
+
+    # @@@DLT_SNIPPET_START markdown_load_back
+        # we reuse the pipeline instance below and load to the same dataset as data
+        pipeline.run([load_info], table_name="_load_info")
+        # save trace to destination, sensitive data will be removed
+        pipeline.run([pipeline.last_trace], table_name="_trace")
+
+        # print all the new tables/columns in
+        for package in load_info.load_packages:
+            for table_name, table in package.schema_update.items():
+                print(f"Table {table_name}: {table.get('description')}")
+                for column_name, column in table["columns"].items():
+                    print(f"\tcolumn {column_name}: {column['data_type']}")
+
+        # save the new tables and column schemas to the destination:
+        table_updates = [p.asdict()["tables"] for p in load_info.load_packages]
+        pipeline.run(table_updates, table_name="_new_tables")
+    # @@@DLT_SNIPPET_END markdown_load_back
+
+    # @@@DLT_SNIPPET_START markdown_notify
+        # check for schema updates:
+        schema_updates = [p.schema_update for p in load_info.load_packages]
+        # send notifications if there are schema updates
+        if schema_updates:
+            # send notification
+            send_slack_message(
+                pipeline.runtime_config.slack_incoming_hook, "Schema was updated!"
+            )
+    # @@@DLT_SNIPPET_END markdown_notify
+
+    # @@@DLT_SNIPPET_START markdown_retry_cm
+        from tenacity import (
```

Comment:
this is really good! but I'd convert it form demo code we used in our walkthrough into something people can copy.
you can base it on `airflow_helper.py` and following code:
```
def log_after_attempt(retry_state: RetryCallState) -> None:
                        if not retry_state.retry_object.stop(retry_state):
                            logger.error("Retrying pipeline run due to exception: %s", retry_state.outcome.exception())

                    try:
                        # retry with given policy on selected pipeline steps
                        for attempt in self.retry_policy.copy(
                            retry=retry_if_exception(retry_load(retry_on_pipeline_steps=self.retry_pipeline_steps)),
                            after=log_after_attempt
                        ):
                            with attempt:
                                logger.info("Running the pipeline, attempt=%s" % attempt.retry_state.attempt_number)
                                load_info = task_pipeline.run(data, table_name=table_name, write_disposition=write_disposition)
                                logger.info(str(load_info))
                                # save load and trace
                                if self.save_load_info:
                                    logger.info("Saving the load info in the destination")
                                    task_pipeline.run([load_info], table_name="_load_info")
                                if self.save_trace_info:
                                    logger.info("Saving the trace in the destination")
                                    task_pipeline.run([task_pipeline.last_trace], table_name="_trace")
                                # raise on failed jobs if requested
                                if self.fail_task_if_any_job_failed:
                                    load_info.raise_on_failed_jobs()
                    finally:
                        # always completely wipe out pipeline folder, in case of success and failure
                        if self.wipe_local_data:
                            logger.info(f"Removing folder {pipeline.working_dir}")
                            task_pipeline._wipe_working_folder()
```
don't do any more loads. just this one. you can also move the slack message and the code that logs updates here
please use logger not prints.